app/models/personal_details.rb

```diff
@@ -1,11 +1,7 @@
 class PersonalDetails < ApplicationRecord
-  # Personal Details
   validates :title, presence: true
   validates :first_name, presence: true
   validates :last_name, presence: true
   validates :date_of_birth, presence: true
   validates :nationality, presence: true
-
-  # Contact Details
```

Comment:
These two comments were probably a sign that this was two models 😄

Reply:
they have been already removed, look at the minus `-` 😉